File: backend/app/src/main/java/com/yagubogu/member/dto/v1/MemberNicknameResponse.java
package com.yagubogu.member.dto.v1;

import com.yagubogu.member.domain.Nickname;

public record MemberNicknameResponse(
        String nickname
) {

    public static MemberNicknameResponse from(final Nickname nickname) {
        return new MemberNicknameResponse(nickname.getValue());
    }
}

File: mobile/composeApp/src/commonMain/kotlin/com/yagubogu/data/dto/response/member/MemberNicknameResponse.kt
package com.yagubogu.data.dto.response.member

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class MemberNicknameResponse(
    @SerialName("nickname")
    val nickname: String, // 유저 닉네임
)
